notes-service/src/main/java/org/exoplatform/wiki/service/impl/NoteServiceImpl.java

```diff
@@ -822,6 +843,16 @@ private void checkToRemoveDomainInUrl(Page note) {
     }
   }
 
+  private boolean canCreateNotes(String authenticatedUser, Space space) throws WikiException {
```

Comment:
Please rename this mthode: it can be "canManageNotes" instead of canCreate

Reply:
done !